pipelines/cellranger/cellranger.wdl

```diff
@@ -117,7 +113,6 @@ task cellranger_count {
         bootDiskSizeGb: boot_disk_size_gb
         disks: "local-disk " + disk_space + " HDD"
         cpu: cpu
-        max_retries: max_retries
```

Comment:
As it turns out, all the uses of `max_retries` here are bogus, as the runtime option is spelled `maxRetries`, so the original code had no effect.